pkg/client/unversioned/extensions.go

```diff
@@ -140,11 +140,7 @@ func setExtensionsDefaults(config *Config) error {
 	}
 	// TODO: Unconditionally set the config.Version, until we fix the config.
 	//if config.Version == "" {
-	gv, err := unversioned.ParseGroupVersion(g.GroupVersion)
-	if err != nil {
-		return err
-	}
-	config.GroupVersion = &gv
+	config.GroupVersion = &g.GroupVersion
```

Comment:
Good catch. Thanks.

Just to confirm my understanding, the reason we make config.GroupVersion a pointer is that we want to use `nil` to tell whether the config.GroupVersion is unset or it's set to the internal version, right?